civis/ml/_model.py

```diff
@@ -531,14 +531,14 @@ class ModelPipeline:
     notifications : dict
         See :func:`~civis.resources._resources.Scripts.post_custom` for
         further documentation about email and URL notification.
-     dependencies : array, optional
```

Comment:
This annoyed the hell out of me (and I think I'm responsible for it too) so thank you!